agreement/events.go

```diff
@@ -43,7 +43,9 @@ type event interface {
 // A ConsensusVersionView is a view of the consensus version as read from a
 // LedgerReader, associated with some round.
 type ConsensusVersionView struct {
-	Err     serializableError
+	_struct struct{} `codec:","`
```

Comment:
Why not omitempty? Does this serialize the (usually nil, I assume) `Err` as a 0 length string?

Reply:
We aren't using omitempty to make encodings as close as possible to what they were before. The one exception to where they wouldn't be identical is `message` struct since we are using `msgp.Raw` there. 

The main reason @cce asked me to remove the `omitempty` hints is for the structs that contain maps since we could accidentally try to insert into a nil map at runtime which would cause a panic. 

I'm happy to add omitemptys on all structs that don't export maps if we want to do that.

I wasn't planning on removing them since there is no noticeable CPU impact as I mentioned and the `persistence.go` implementation only ever stores a single row in the DB and with WAL enabled on the DB the small amount of IO increase here shouldn't be a concern either.